moon/bulk_test.sh

```diff
@@ -99,17 +99,17 @@ esac
 
 now=$(date +"%m%d%Y_%H%M")
 log_dir="${LOG_DIR_PREFIX}/bulklogs_${now}"
-mkdir ${log_dir}
+mkdir "${log_dir}"
```

Comment:
are you using spaces in path now?

Reply:
yeah, named my external drive with a space in its name, changed that later as this wasn't the only code where it didn't work (lidarview, ahem)